specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,1170 @@
+
+        // Add a selection item to the AppNotificationComboBox if InputType is AppNotificationComboBox. The id parameter is used to retrieve the user input when the app is activated
+        // and content is the displayed text for the item.
+        AppNotificationComboBox AddItem(String id, String content);
+
+        // Adds a title to display on top
+        AppNotificationComboBox SetTitle(String value);
+
+        // Sets the default selection to be displayed by the AppNotificationComboBox
+        AppNotificationComboBox SetSelectedItem(String id);
+
+        // Retrieves the XML content of the input.
+        String GetXml();
+    };
+
+    runtimeclass AppNotificationProgressBar
+    {
+        // AppNotificationProgressBar binds to AppNotificationProgressData so the AppNotification will
+        // receive every update to the status and value. In the WinAppSDK, these binding
+        // values are static, so developers won't need to define these binding values
+        // themselves.
+        AppNotificationProgressBar();
+
+        // Adds a static title to the progress bar
+        AppNotificationProgressBar StaticTitleForLabel(String value);
+
+        // Adds the title binding value to be used with AppNotificationProgressData
+        AppNotificationProgressBar BindTitleForLabel();
+
+        // Adds the valueStringOverride value.
+        AppNotificationProgressBar BindValueStringOverride();
+
+        // Retrieves the XML content of the AppNotificationProgressBar
+        String GetXml();
+    };
+
+    enum MSWinSoundEvent
+    {
+        Default,
+        IM,
+        Mail,
+        Reminder,
+        SMS,
+        Alarm,
+        Alarm2,
+        Alarm3,
+        Alarm4,
+        Alarm5,
+        Alarm6,
+        Alarm7,
+        Alarm8,
+        Alarm9,
+        Alarm10,
+        Call,
+        Call2,
+        Call3,
+        Call4,
+        Call5,
+        Call6,
+        Call7,
+        Call8,
+        Call9,
+        Call10,
+    };
+
+    enum AppNotificationScenario
+    {
+        Default, // The normal AppNotification behavior. The AppNotification appears for a short duration, and then automatically dismisses into Notification Center.
+        Reminder, // The notification will stay on screen until the user dismisses it or takes action.
+        Alarm,    // Alarms behave like Reminder, but alarms will additionally loop audio with a default alarm sound.
+        IncomingCall, // Incoming call notifications are displayed pre-expanded in a special call format and stay on the user's screen till dismissed.
+        Urgent, // Important notifications allow users to have more control over what 1st party and 3rd party apps can send them high-priority AppNotifications (urgent/important) that can break through Focus Assist.
+    };
+
+    enum AppNotificationDuration
+    {
+        Short, // Default value. AppNotification appears for a short while and then goes into Notification Center.
```

Comment:
"Short" doesn't mean "short". It just means "standard duration". Should be "Default".

Reply:
No wait, maybe Short does mean Short. I think there are three possible durations:
* Default
* Short
* Long

I infer this from the schema which says `duration? = "long" | "short"`, so the duration can be omitted (default), or explicitly made short or long.

So we should have three values for this enumeration.

Reply:
Default & Long